scripts/build-image/Dockerfile

```diff
@@ -4,12 +4,12 @@ WORKDIR /tmp
 
 # Install Helm
 ENV HELM_VERSION=2.13.0
-RUN curl -sLO https://kubernetes-helm.storage.googleapis.com/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
+RUN curl -sLO https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
```

Comment:
Why is this change required?

Reply:
I was getting a 403 for that url when trying to run the build locally. Not sure if this file is used in the build or not, if it is maybe cloud infrastructure is whitelisted. Happy to back it out if you'd like or only use that url if the first one fails.